os/se/ameba/security_ameba_wrapper.c

```diff
@@ -270,6 +272,18 @@ static void* my_calloc(size_t nelements, size_t elementSize)
 	return ptr;
 }
 
+static uint32_t rtl_index_convt(uint32_t key_idx)
+{
+	uint32_t ret_key_idx = KEY_STORAGE_INDEX_MAX;
+
+	/* Because Ameba SE storing Key in array, array start from Zero */
+	if (key_idx != 0) {	/* Index not Zero */
+		/* Valid  Inded are 0 - 31, Total 32 Index slot */
```

Comment:
Please fix `Inded` to `Index`.

Reply:
shall we use macro for this function?

Reply:
So I prefer to use as static function, will it cause any problem?